README.rst

```diff
@@ -796,6 +774,34 @@ Get command details
 
     GET /api/v1/connection/device/{device_pk}/command/{command_pk}
 
+Get device coordinates
```

Comment:
I guess here we want to make the REST API section of the README more consistent by replacing the extended description for the given endpoint with a more brief description similar to the description of all the other endpoints, right? I mean this is an intended change, right?